packages/website/src/bin/page-generator/index.js

```diff
@@ -43,44 +39,32 @@ const scanAndGenerate = (docs, docsPath, generatorConfig, name) => {
           pagePath: path.join(doc.id, child.id),
         })),
       };
-      if (readme) {
-        generateDocsHomePage(
-          path.join(pagePath, 'index.js'),
-          { ...docData, ...pageData },
-          generatorConfig,
-          'Documents',
-        );
 
+      generateDocsHomePage(
+        path.join(pagePath, 'index.js'),
+        { ...docData, ...pageData },
+        generatorConfig,
+        'Documents',
+      );
+
+      const readme = doc.children.find(
+        c => c.path && c.path.toLowerCase().match(/readme\.md$/),
+      );
+      if (readme) {
```

Comment:
What's with this inversion?

Reply:
By inversion, do you mean placing `generateDocsHomePage` above the conditional? It's being called both inside/outside the condition so it can be hoisted up.